src/components/ProgressBars/LinearProgressBar/__stories__/linearProgressBar.stories.scss

```diff
@@ -31,7 +29,8 @@
 
   &_without-bg {
     .linear-progress-bar__container {
-      @include theme-prop(background-color, primary-text-on-primary);
+      // TODO check
```

Comment:
why not semantic prop :( ! and let's check and then remove the todo

Reply:
**TODO needs to be revisited, as the point was in displaying story without progress bar background color using css override, which is not working anymoew**

Reply:
Removed completely as an example